pages/download.html

```diff
@@ -309,6 +309,19 @@ <h3>{% trans "Linux / Source Code" %}</h3>
   </div>
 </div>
 
+<div id="snapshots" class="docs-section container">
+  <h2><a class="headline-link" href="#snapshots">{% blocktrans %}Development Snapshots{% endblocktrans %}</a></h2>
+  <p>
+    {% blocktrans with source_link="https://github.com/mixxxdj/mixxx" windows_snapshot_link="https://downloads.mixxx.org/builds/appveyor/cmake_build/?C=M;O=A" macos_snapshot_link="https://downloads.mixxx.org/builds/travis/main/?C=M;O=A" ubuntu_snapshot_link="https://downloads.mixxx.org/builds/travis/main/?C=M;O=A" %}Want to test and experiment with the latest changes?
```

Comment:
```suggestion
    {% blocktrans with source_link="https://github.com/mixxxdj/mixxx" windows_snapshot_link="https://downloads.mixxx.org/builds/appveyor/cmake_build/?C=M;O=D" macos_snapshot_link="https://downloads.mixxx.org/builds/travis/main/?C=M;O=D" ubuntu_snapshot_link="https://downloads.mixxx.org/builds/travis/main/?C=M;O=D" %}Want to test and experiment with the latest changes?
```

Whoops, these should be in descending order of date modified.